README.md

```diff
@@ -44,30 +44,40 @@ This boilerplate has a [wiki](https://github.com/Shift3/boilerplate-client-angul
 
 The AWS configuration **for the sandbox** is handled by Terraform. Terraform needs the AWS credentials which developers should already have or can access through Zoho Vault. The Terraform configuration is separated into modules for each cloud service it sets up.
 
-Terraform also needs the project secrets saved in `project/terraform/main/<environment-name>/terraform.tfvars` with the following structure:
+Terraform also needs the project secrets saved in `project/terraform/terraform.tfvars` with the following structure:
 
 ```
-profile = "shift3"
+aws_profile = "shift3"
 
-region = "us-west-2"
+aws_region = "us-west-2"
 
 web_domain_name = ""
 
 ```
 
 | Secret          |                                                                                             Note |
 | :-------------- | -----------------------------------------------------------------------------------------------: |
-| profile         |                              This must match the AWS credentials name on the development machine |
-| region          |                                                                      This is usually `us-west-2` |
+| aws_profile     |                              This must match the AWS credentials name on the development machine |
+| aws_region      |                                                                      This is usually `us-west-2` |
 | web_domain_name | This will be the web domain name for the project, an example may be: `example.shift3sandbox.com` |
 
-Once this is completed, navigate to the terraform/main/<environment-name> folder in your terminal and run the following commands:
+Terraform also needs to update the following key value within `main.tf` by replacing <project-name> and <environment> with actual values:
```

Comment:
Nice explanation here.

Reply:
Thanks.that's right I found there is a need to have this clarification since many of us forgot to update it before.